src/main/java/io/aiven/kafka/connect/opensearch/BulkProcessor.java

```diff
@@ -410,6 +432,15 @@ private void handleVersionConflict(final BulkItemResponse bulkItemResponse) {
                                     + " records. Ignoring and will keep an existing record. Error was {}",
                             batchId, batch.size(), bulkItemResponse.getFailureMessage());
                     break;
+                case REPORT:
+                    final String errorMessage =
+                            String.format("Encountered a version conflict when executing batch %s of %s"
+                                            + " records. Reporting to DLQ and will keep an existing record."
```

Comment:
This will not necessarily cause anything to be sent to a DLQ topic, so we shouldn't assume that in our log message:

```suggestion
                                            + " records. Reporting this error to the Connect runtime and will keep an existing record."
```